app/assets/javascripts/controllers/reconfigure/reconfigure_form_controller.js

```diff
@@ -271,6 +293,41 @@ ManageIQ.angular.app.controller('reconfigureFormController', ['$http', '$scope',
     vm.updateNetworkAdaptersAddRemove();
   };
 
+  vm.editExistingNetworkAdapter = function(thisNetworkAdapter) {
+    thisNetworkAdapter.vLan_edit_requested = null;
+    thisNetworkAdapter.add_remove = 'update';
+    thisNetworkAdapter.originalVLan = thisNetworkAdapter.vlan;
+    vm.updateNetworkAdaptersAddRemove();
+  };
+
+  vm.cancelChangeExistingNetworkAdapter = function(thisNetworkAdapter) {
+    thisNetworkAdapter.vLan_edit_requested = null;
+    thisNetworkAdapter.vlan = thisNetworkAdapter.originalVLan;
+    thisNetworkAdapter.add_remove = '';
+    vm.updateNetworkAdaptersAddRemove();
+  };
+
+  vm.confirmEditSelectedNetworkAdapter = function(networkAdapter) {
+    angular.forEach(vm.reconfigureModel.vmNetworkAdapters, function(nap) {
+      if (nap.name == networkAdapter.name){
+        nap.originalVLan = networkAdapter.vlan
+        nap.vlan = networkAdapter.vLan_edit_requested,
+        nap.add_remove = 'edit'
+      }
+    });
+    vm.updateNetworkAdaptersAddRemove();
+  };
+
+  vm.checkEditNetworkAdapter = function() {
+    checkEditNetworkAdapterRes = false;
```

Comment:
variables should be initialized with var

Reply:
yes, will change this

Reply:
Fixed